waspc/data/Generator/templates/react-app/src/config.js

```diff
@@ -1,9 +0,0 @@
-import { stripTrailingSlash } from "./universal/url";
```

Comment:
This template file wasn't used anywhere, so I removed it. It's a leftover from migrating to the SDK.